public/js/components/AtomEdit/CustomEditors/TimelineFields/TimelineItem.js

```diff
@@ -3,30 +3,74 @@ import {ManagedForm, ManagedField} from '../../../ManagedEditor';
 import FormFieldTextInput from '../../../FormFields/FormFieldTextInput';
 import FormFieldsScribeEditor from '../../../FormFields/FormFieldScribeEditor';
 import FormFieldDateTextInput from '../../../FormFields/FormFieldDateTextInput';
+import FormFieldRadioButtons from '../../../FormFields/FormFieldRadioButtons';
+import FormFieldCheckbox from "../../../FormFields/FormFieldCheckbox";
 
 export class TimelineItem extends React.Component {
   static propTypes = {
     fieldLabel: PropTypes.string,
     fieldName: PropTypes.string,
     fieldValue: PropTypes.shape({
-      title: PropTypes.string,
-      date: PropTypes.number,
-      body: PropTypes.string
+      title: PropTypes.string.isRequired,
+      date: PropTypes.number.isRequired,
+      toDate: PropTypes.number,
+      body: PropTypes.string,
+      dateFormat: PropTypes.string
     }),
     fieldPlaceholder: PropTypes.string,
     onUpdateField: PropTypes.func,
     onFormErrorsUpdate: PropTypes.func
   };
 
+    state = {
+        dateRangeRequired: typeof this.props.fieldValue === PropTypes.shape({
```

Comment:
what's happening here, is it checking for the existence of `toDate`?

Reply:
Yes it is. On first render, `fieldValue` is undefined so this is why `this.props.fieldValue.toDate` doesn't work and we require a more verbose solution

Reply:
have you tried `this.props.fieldValue && 'toDate' in this.props.fieldValue`?

Reply:
It works but produces a few warnings. I think I'm going to stick with the above for now.